### ISSUE TITLE
UI Issue with font size and buttons overlap

### ISSUE THREAD

**Issue by a-lang** [GoogleChromeLabs/text-app#475]
After updating Chromium to 76.0.3809.100 the app got an issue with UI appearence as the attached screenshot.

The OS is Peppermint 10(Ubuntu 18.04 based), 32-bit

![text-issue](https://user-images.githubusercontent.com/11038458/63570240-07a95f80-c5af-11e9-9f82-5fda7a95d9ec.png)

**Comment by BugsNash** [comment]
Not able to repro this on Chrome OS version 76.0.3809.102 nor on Debian on Chrome 76.0.3809.100

What is the language you're viewing in?

**Comment by BugsNash** [comment]
It looks like the icon font isn't loading. Can you reliably reproduce this?

**Comment by a-lang** [comment]
> Not able to repro this on Chrome OS version 76.0.3809.102 nor on Debian on Chrome 76.0.3809.100
> 
> What is the language you're viewing in?

Chrome is Traditional Chinese

**Comment by BugsNash** [comment]
Can't repro in chrome os when viewing in traditional chinese 

![Screenshot 2019-09-06 at 6 51 00 AM](https://user-images.githubusercontent.com/6046079/64379991-6ec91880-d073-11e9-84ec-df804100ab75.png)

I can't seem to get language preferences in chrome browser to work on linux, but I can't repro this in English on linux. 

Are you still seeing this problem? Can you try uninstalling and reinstalling the text app?

**Comment by a-lang** [comment]
Yes, I've reinstalled the app for several times, still got the same issue.
Attached full screenshot on my desktop, if any logs is helpful to debug just let me know.
![text-issue2](https://user-images.githubusercontent.com/11038458/64393762-8f01d300-d085-11e9-806f-a855b1215f05.png)

**Comment by BugsNash** [comment]
Could be a problem related to chrome apps being deprecated on linux. I'm not sure why the icon font wouldn't be loading even after reinstall. The fonts are included in the package.

**Comment by a-lang** [comment]
The OS is 32-bit, not sure if it's a problem to the issue.
